cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -332,6 +332,8 @@ declare namespace Deno {
    * ```
    *
    * Requires `allow-sys` permission.
+   * 
+   * On Windows there is no API available to retrieve this information and this method returns 0.
```

Comment:
```suggestion
   * On Windows there is no API available to retrieve this information and this method returns `[ 0, 0, 0 ]`.
```

Maybe more precisely this?